go-magic-link/README.md

```diff
@@ -0,0 +1,59 @@
+# Go-admin-portal-example
+An example Golang application demonstrating how to use the [WorkOS Golang SDK](https://github.com/workos-inc/workos-go) so your customers can access the WorkOS Admin Portal from your application.
+
+
+## Go Project Setup
+
+1. In your CLI, navigate to the directory into which you want to clone this git repo.
+   ```bash
+   $ cd ~/Desktop/
+   ```
+
+2. Clone the example repo using your preferred secure method (HTTPS or SSH).
+   ```bash
+   # HTTPS
+   $ git clone https://github.com/workos-inc/Go-Example-Applications.git
+   ```
+
+   or
+
+   ```bash
+   # SSH
+   $ git clone git@github.com:workos-inc/Go-Example-Applications.git
+   ```
+
+3. Navigate to the cloned repo and to the admin portal:
+   ```bash
+   $ cd Go-Example-Applications/go-magic-link
+   ```
+
+4. Obtain and make note of the following values. In the next step, these will be set as environment variables.
+   - Your [WorkOS API key](https://dashboard.workos.com/api-keys)
+
+5. Add this to your APIKey variable.
+
+## Start the server
+
+11. The final setup step is to start the server.
+   ```bash
+   go run main.go
+   ```
+
+   You'll know the server is running when you see no errors in the CLI, and output similar to the following is displayed:
+
+   ```bash
+   launching admin portal demo with configuration: {Addr::8000 Domains:}
+   ```
+
+   Navigate to `localhost:8000` in your web browser to view the homepage of the Admin Portal example app. Enter the name of the new Organization to be created and the names of all of the Organization's associated domains.
+
+   - The Organization must be a new Organization that doesn't yet exist in your WorkOS dashboard
+   - The domains should be entered as space-separated values, e.g. "domain1.com domain2.com domain3.com"
+
+   Then, click the buttons to either create a new SSO connection or a new Directory Sync connection.
+   
+   Hooray!
```

Comment:
The user will also need to set the default redirect URI as "http://localhost:8000/success" in their WorkOS Dashboard Configuration tab. Can you add a blurb about that in the readme?